.github/workflows/notify-release.yml

```diff
@@ -0,0 +1,26 @@
+name: PRs not yet released checker
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs on Mondays at 8AM PST (15 UTC)
```

Comment:
Thank you for commenting. I can't grok these without looking up what things mean each time haha.